README.md

```diff
@@ -195,6 +195,42 @@ allowed inbound rules in the databases' security groups. If the databases do not
 analyze what is the proper networking configuration to allow connectivity from the EC2
 instances to the protected databases.
 
+#### Enable the S3 File Browser
```

Comment:
This section should be in the module itself.

Reply:
More precisely, move this to a new file `docs/s3-browser.mdx` and add the link to the `Advanced` section in the [actual Terraform module](https://github.com/cyralinc/terraform-aws-sidecar-ec2/).